src/login/login.component.tsx

```diff
@@ -25,6 +28,12 @@ export default function Login(props: LoginProps) {
       "https://implementation-assets.sfo2.digitaloceanspaces.com/openmrs_logo_white_large.png"
   };
 
+  React.useEffect(() => {
+    if (document.activeElement !== usernameInputRef.current) {
+      passwordInputRef.current.focus();
+    }
+  }, [showPassword, passwordInputRef.current, usernameInputRef.current]);
+
   const input = css`
     width: 100%;
     padding: 10px 5px;
```

Comment:
Why do use px here? In most other cases, you seem to recommend using REM.

Reply:
I didn't change this line of code -- it existed from when fatma and i implemented this a couple months ago.

Yes, we should be using rems, though.